tests/Unit/PrestaShopBundle/ApiPlatform/DomainSerializerTest.php

```diff
@@ -83,5 +86,42 @@ public function getExpectedDenormalizedData()
                 [1]
             ),
         ];
+
+        $editCartRuleCommand = new EditCartRuleCommand(1);
+        $editCartRuleCommand->setDescription('test description');
+        $editCartRuleCommand->setCode('test code');
+        $editCartRuleCommand->setMinimumAmount('10', 1, true, true);
+        $editCartRuleCommand->setCustomerId(1);
+        $editCartRuleCommand->setLocalizedNames([1 => 'test1', 2 => 'test2']);
+        $editCartRuleCommand->setHighlightInCart(true);
+        $editCartRuleCommand->setAllowPartialUse(true);
+        $editCartRuleCommand->setPriority(1);
+        $editCartRuleCommand->setActive(true);
+        $editCartRuleCommand->setValidityDateRange(new \DateTimeImmutable('2023-08-23'), new \DateTimeImmutable('2023-08-25'));
+        $editCartRuleCommand->setTotalQuantity(100);
+        $editCartRuleCommand->setQuantityPerUser(1);
+        $editCartRuleCommand->setCartRuleAction(new CartRuleAction(true));
+        yield [
+            [
+                'cartRuleId' => 1,
+                'description' => 'test description',
+                'code' => 'test code',
+                'minimumAmount' => ['minimumAmount' => '10', 'currencyId' => 1, 'taxIncluded' => true, 'shippingIncluded' => true],
```

Comment:
```suggestion
                'minimumAmount' => ['minimumAmount' => 10.15, 'currencyId' => 1, 'taxIncluded' => true, 'shippingIncluded' => true],
```
This should work ith a float value (or maybe both types?)

Reply:
the setter take a string as this parameter